packages/react-devtools-timeline/src/Timeline.js

```diff
@@ -62,17 +62,17 @@ export function Timeline(_: {||}) {
 
   return (
     <div className={styles.Content} ref={ref}>
-      {timelineData ? (
+      {file ? (
         <Suspense fallback={<ProcessingData />}>
-          <DataResourceComponent
-            dataResource={timelineData}
+          <FileLoader
```

Comment:
i assume all of this is just renaming but preserves the same behavior?

Reply:
Preserves the same behavior, yes. Not quite just  rename- but an updating of the code here to better match the other Suspense caches.